### PR TITLE
Updated link in docs/sql/tutorials/mindsdb-superset-snowflake.mdx

### PR DESCRIPTION
**Pull request by ananyabhat29**
## Description

Updated link in docs/sql/tutorials/mindsdb-superset-snowflake.mdx

**Fixes** #6410 

## Checklist:

- [x] I have updated the documentation, or created issues to update them.

### CODE REVIEW

**Comment by ananyabhat29** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by martyna-mindsdb** [comment]
recheckcla

**Comment by mindsdbadmin** [comment]
**CLA Assistant Lite bot:** <br/>Thank you for your submission, we really appreciate it. Like many open-source projects, we ask that you sign our [Contributor License Agreement](https://github.com/mindsdb/mindsdb_native/blob/stable/assets/contributions-agreement/individual-contributor.md) before we can accept your contribution. You can sign the CLA by just posting a Pull Request Comment same as the below format.<br/>
   - - -
   I have read the CLA Document and I hereby sign the CLA
   - - -
   <sub>You can retrigger this bot by commenting **recheck** in this Pull Request</sub>

**Comment by martyna-mindsdb** [comment]
recheckcla

File: docs/sql/tutorials/mindsdb-superset-snowflake.mdx [hunk 1]
```diff
@@ -174,8 +174,8 @@ The combination of MindsDB and your database covers all the phases of the ML lif
 
 
 MindsDB provides easy-to-use predictive models through AI Tables. You can create these predictive models using SQL statements and feeding the input data. Also, you can query them the same way you query a table. The easiest way to get started with Superset is with the free tier for [Preset Cloud](https://preset.io/product/), a hassle-free and fully hosted cloud service for Superset.
-
-We encourage you to try some predictions with your own data, so please sign up for a [free MindsDB cloud account](https://cloud.mindsdb.com/signup) and if you need any help with MindsDB, feel free to ask our [Slack](https://join.slack.com/t/mindsdbcommunity/shared_invite/zt-o8mrmx3l-5ai~5H66s6wlxFfBMVI6wQ) and [Github](https://github.com/mindsdb/mindsdb/discussions) communities.
+[Slack](https://mindsdb.com/joincommunity)
+We encourage you to try some predictions with your own data, so please sign up for a [free MindsDB cloud account](https://cloud.mindsdb.com/signup) and if you need any help with MindsDB, feel free to ask our  and [Github](https://github.com/mindsdb/mindsdb/discussions) communities.
```
Comment (martyna-mindsdb) [inline docs/sql/tutorials/mindsdb-superset-snowflake.mdx:178]: You moved the link to the previous line, which is not as in the instructions.

Please bring the link back to the sentence. The sentence should be as below:

We encourage you to try some predictions with your own data, so please sign up for a \[free MindsDB cloud account](https://cloud.mindsdb.com/signup) and if you need any help with MindsDB, feel free to ask **our \[Slack](https://mindsdb.com/joincommunity)** and \[Github](https://github.com/mindsdb/mindsdb/discussions) communities.

**Comment by martyna-mindsdb** [comment]
@ananyabhat29 
Please see the review comment above.

Please implement the change and push it to this PR.

**Comment by ananyabhat29** [comment]
@martyna-mindsdb Sorry for the confusion. I have corrected it and pushed a commit to this PR. Please review.

**Pull request merged**